string name
int32 destination
int32 product
int32 handle
int32 handlePlatform
int32 zone
